Client/sdk/game/CTaskManager.h

```diff
@@ -41,6 +41,13 @@ enum
     ABORT_PRIORITY_IMMEDIATE
 };
 
+enum taskType
```

Comment:
be consistent with another code. `eTaskType`. It's not a part of hungarian notation

Reply:
so what letter e reference to?

Reply:
`e` -> `enum`

Reply:
enumeration

Reply:
so we can use hungarian notation with enums?
I mean I don't see any benefits of using that format

Reply:
> so we can use hungarian notation with enums? I mean I don't see any benefits of using that format

Using e for enumartion is not considered Hungarian notation, it's a different principle and a practiced used in all code

Reply:
I do not see the point of renaming this

Reply:
It's too trivial to block this